common/decisions/00_protestant_reformation_decision.txt

```diff
@@ -0,0 +1,60 @@
+protestant_reformation_decision = {
+	picture = {
+		reference = "gfx/interface/illustrations/decisions/decision_dynasty_house.dds"
+	}
+	desc = protestant_reformation_decision_desc
+	selection_tooltip = protestant_reformation_decision_tooltip
+	decision_group_type = major
+
+	ai_check_interval = 120
+
+	is_shown = {
+		root = {
+			OR = {
+				culture = culture:saxon
+				culture = culture:bavarian
+				culture = culture:franconian
+				culture = culture:swabian
+				culture = culture:polabian
+			}
+		}
+
+		# exists = faith.religious_head
+	}
+	
+	is_valid = {
+		faith.religious_head = {
+			opinion = {
+				target = root
+				value < -100
+			}
+		}
+
+		capital_county = {
+			development_level => 35
+		}
+	}
```

Comment:
Overall looks great - only thing missing for the is_valid is the requirement that this can only happen within German lands.
This will be a custom region, include any Central Germanic cultures for the custom region - they must be within the region (not control it, be within it)

Reply:
Here are the Changes:

`	is_valid = {
		faith.religious_head = {
			opinion = {
				target = root
				value < -100
			}
		}

		capital_county = {
			development_level > 35
		}

		root = {
			any_held_title = {
				title_province = { geographical_region = central_germanic_lands }
			}
		}
	}`